week-1/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,42 @@
+function percentage(Students, Mentors) {
+  var studentPercentage = Math.round(100 * Students / (Students + Mentors));
+  return studentPercentage;
+}
+
+function text(Students, Mentors) {
+  var findPercentage = percentage(Students, Mentors);
+  var mentorPercentage = 100 - findPercentage;
+  var message =
+    "percentsge of students is " +
+    findPercentage +
+    "% and percentage of mentors is " +
+    mentorPercentage +
+    "%";
+  return message;
+}
+
+console.log(text(15, 8));
+
+//...........Exercise2......
+
 var mentor1 = "Daniel";
 var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function spell(name) {
```

Comment:
I think the exercise is a bit unclear about what they're asking you to do - they say to create a function that returns the name all in caps, but then the whole output is in capitals. I think in your case you don't need this function - would you agree?

Reply:
Yes, you are right because of the examples at the end of the instruction, I thought that greeting should also be in the capital.